orion/handlers/http/http.go

```diff
@@ -33,24 +32,9 @@ func (h *httpHandler) httpHandler(resp http.ResponseWriter, req *http.Request, s
 	ctx := utils.StartNRTransaction(req.URL.Path, req.Context(), req, resp)
 	ctx = loggers.AddToLogContext(ctx, "transport", "http")
 	var err error
-	defer func(resp http.ResponseWriter, ctx context.Context, t time.Time) {
-		// panic handler
-		if r := recover(); r != nil {
-			writeResp(resp, http.StatusInternalServerError, []byte("Internal Server Error!"))
```

Comment:
what is the HTTP response in case of panic ?

Reply:
It will also be handled by PanicRecoverInterceptor I just added because of this line: https://github.com/carousell/Orion/blob/master/orion/handlers/http/http.go#L156

one of the targets in this PR is to make both HTTP and GRPC panics are handled in one place.